### PR TITLE
Update Personal Tasks

### PR DESCRIPTION
**Pull request by commy2**

### CODE REVIEW

**Comment by commy2** [comment]
double check:

- [ ] Boss Helix Inconsistencies

File: Patch104pZH/Design/Tasks/commy2_tasks.txt [hunk 1]
```diff
@@ -169,43 +169,43 @@ https://github.com/commy2/zerohour/issues/52  [DONE][NPROJECT]        Boss Base
...
 https://github.com/commy2/zerohour/issues/50  [DONE][NPROJEC!]        Demo General Vehicle Destruction Effect Glitches
 https://github.com/commy2/zerohour/issues/49  [DONE][NPROJEC!]        Demo General Infantry Play Terrorist Sound Effect When Killed
-https://github.com/commy2/zerohour/issues/48  [MAYBE][NPROJECT]       Demo General Scud Launcher With Demolitions Upgrade Deals Full Damage When Destroyed
+https://github.com/commy2/zerohour/issues/48  [DONE][NPROJECT]        Demo General Scud Launcher With Demolitions Upgrade Deals Full Damage When Destroyed
 https://github.com/commy2/zerohour/issues/47  [DONE]                  Demo General Battle Bus Is Deleted Without Dealing Damage When Suicided
 https://github.com/commy2/zerohour/issues/46  [DONE][NPROJEC!]        Technical Has To Recenter Turret To Suicide
 https://github.com/commy2/zerohour/issues/45  [DONE]                  Combat Bike Has Disabled Suicide Ability Button
-https://github.com/commy2/zerohour/issues/44  [MAYBE][NPROJECT]       Terrorist And Bomb Truck Missing Suicide Ability
+https://github.com/commy2/zerohour/issues/44  [WONTFIX][NPROJECT]     Terrorist And Bomb Truck Missing Suicide Ability
```
Comment (commy2) [inline Patch104pZH/Design/Tasks/commy2_tasks.txt]:
```suggestion
https://github.com/commy2/zerohour/issues/44  [REJECTED][NPROJECT]    Terrorist And Bomb Truck Missing Suicide Ability
```

**Review by xezon — APPROVED** [review]

File: Patch104pZH/Design/Tasks/commy2_tasks.txt [hunk 2]
```diff
@@ -169,43 +169,43 @@ https://github.com/commy2/zerohour/issues/52  [DONE][NPROJECT]        Boss Base
 https://github.com/commy2/zerohour/issues/51  [DONE][NPROJEC!]        Demo General Worker On Bike Missing Demo Generals Bike Destruction Effect
 https://github.com/commy2/zerohour/issues/50  [DONE][NPROJEC!]        Demo General Vehicle Destruction Effect Glitches
 https://github.com/commy2/zerohour/issues/49  [DONE][NPROJEC!]        Demo General Infantry Play Terrorist Sound Effect When Killed
-https://github.com/commy2/zerohour/issues/48  [MAYBE][NPROJECT]       Demo General Scud Launcher With Demolitions Upgrade Deals Full Damage When Destroyed
+https://github.com/commy2/zerohour/issues/48  [DONE][NPROJECT]        Demo General Scud Launcher With Demolitions Upgrade Deals Full Damage When Destroyed
```
Comment (commy2) [inline Patch104pZH/Design/Tasks/commy2_tasks.txt]:
```suggestion
https://github.com/commy2/zerohour/issues/48  [MAYBE][NPROJECT]       Demo General Scud Launcher With Demolitions Upgrade Deals Full Damage When Destroyed
```

File: Patch104pZH/Design/Tasks/commy2_tasks.txt [hunk 3]
```diff
@@ -57,34 +57,34 @@ https://github.com/commy2/zerohour/issues/173 [DONE]                  Humvee Wit
...
 https://github.com/commy2/zerohour/issues/162 [IMPROVEMENT][NPROJECT] Boss General Has Some Conflicting Hotkeys
-https://github.com/commy2/zerohour/issues/161 [MAYBE][NPROJECT]       Terrorist And Terror Bike Have To Force-Fire Ground To Detonate
-https://github.com/commy2/zerohour/issues/160 [MAYBE]                 Slow Units Can Outrun Comanche Missiles
+https://github.com/commy2/zerohour/issues/161 [REJECTED][NPROJECT]    Terrorist And Terror Bike Have To Force-Fire Ground To Detonate
+https://github.com/commy2/zerohour/issues/160 [WONTFIX]               Slow Units Can Outrun Comanche Missiles
 https://github.com/commy2/zerohour/issues/159 [DONE][NPROJECT]        Boss Nationalism Upgrade Changes Tooltip After Land Mines Upgrade
 https://github.com/commy2/zerohour/issues/158 [DONE][NPROJECT]        Boss Patriot Battery Has Unique Stop Button Position
-https://github.com/commy2/zerohour/issues/157 [MAYBE][NPROJECT]       Boss General Has Strange Selection Of Available Upgrades
+https://github.com/commy2/zerohour/issues/157 [DONE][NPROJECT]        Boss General Has Strange Selection Of Available Upgrades
```
Comment (commy2) [inline Patch104pZH/Design/Tasks/commy2_tasks.txt]:
```suggestion
https://github.com/commy2/zerohour/issues/157 [MAYBE][NPROJECT]       Boss General Has Strange Selection Of Available Upgrades
```

**Pull request merged**